gammapy/astro/population/simulate.py

```diff
@@ -1,7 +1,8 @@
 # Licensed under a 3-clause BSD style license - see LICENSE.rst
 """Simulate source catalogs.
 """
-from __future__ import absolute_import, division, print_function, unicode_literals
+from __future__ import (absolute_import, division, print_function,
+                        unicode_literals)
```

Comment:
Please leave this as single line. I know it's different in Astropy, but we have it as 1 line 100x in Gammapy.